pom.xml

```diff
@@ -0,0 +1,208 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>Services</groupId>
+  <artifactId>ServicesUtilities</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>ServicesUtilities</name>
+  <description>Simple utilities for UCLA Services Team projects</description>
+  
+  <licenses>
+    <license>
+      <name>The 3-Clause BSD License</name>
+      <url>https://opensource.org/licenses/BSD-3-clause</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>DRickard</id>
+      <name>David Rickard</name>
+      <email>drickard1967@library.ucla.edu</email>
+      <organization>UCLA Library</organization>
+      <organizationUrl>http://github.com/uclalibrary</organizationUrl>
+      <timezone>America/Los_Angeles</timezone>
+    </developer>
+  </developers>
```

Comment:
I'm of the mind that you might as well as the rest of us too... we haven't contributed code yet, but between reviewing and, if we view this as a dropping place for general utilities that we create, we'll have our hands in it before too long.